sdk/dapp-kit/src/components/AccountDropdownMenu.tsx

```diff
@@ -46,7 +48,7 @@ export function AccountDropdownMenu({ currentAccount }: AccountDropdownMenuProps
 								className={clsx(styles.menuItem, styles.switchAccountMenuItem)}
 								onSelect={() => switchAccount({ account })}
 							>
-								<Text mono>{formatAddress(account.address)}</Text>
+								<Text mono>{account.label ?? formatAddress(account.address)}</Text>
```

Comment:
should we similarly try to use the ENS name for other accounts in the dropdown?

Reply:
I was a little worried about doing n lookups for this list... maybe? I can be convinced either way.  There's no bulk load method

Reply:
Do these components actually get mounted when the UI is closed? If yes, I think it's probably fine to do N lookups when the dropdown is opened, but probably not beforehand. (although this is not great UI because it'll double-render, once with the address, then with the suins name when it loads). GraphQL hopefully solves this?